streamer/packager_node.py

```diff
@@ -151,6 +151,9 @@ def _setup_stream(self, stream: OutputStream) -> str:
           SINGLE_SEGMENT[stream.type],
           dir=self._segment_dir)
 
+    if stream.is_dash_only():
+      dict['dash_only'] = str(1)
```

Comment:
Style nit: I think you should use '1' instead of str(1).

Reply:
I originally thought that too, but the dictionary is looking for string keys. This is the output when using '1' rather than 'str(1)':
![image](https://user-images.githubusercontent.com/38890251/122470001-6bfe9580-cf72-11eb-9270-506bb9bc863a.png)
The dictionary is used to populate the Packager command, which requires string type for the concatenation.

Reply:
What I'm saying is that you should use a string literal instead of a cast:

```py
      dict['dash_only'] = '1'
```

Reply:
Oh shoot, sorry! That was my misinterpretation. It has been fixed now.